framework/src/org/checkerframework/framework/util/defaults/QualifierDefaults.java

```diff
@@ -525,10 +526,13 @@ private void applyDefaultsElement(final Element annotationScope, final Annotated
                 // but it doesn't work correctly and tests fail.
                 // (That whole @FromStubFile and @FromByteCode annotation
                 // logic should be replaced by something sensible.)
-            if ((ElementUtils.isElementFromByteCode(annotationScope) &&
+        	SourceChecker checker = atypeFactory.getContext().getChecker();
```

Comment:
Sorry about that - I was used to RBCommons giving me a big red warning whenever I uploaded a patch with tabs or trailing spaces due to Eclipse resetting my editor configuration without my knowledge. I'll pay attention to this in the future.